### PR TITLE
Add Support for go-changelog / changelog-build

### PR DESCRIPTION
**Pull request by nwchandler**
This PR adds support for the changelog-build command in order to improve the process for creation of changelogs / release notes going forward.

### CODE REVIEW

**Review by aliciaclark1066 — COMMENTED** [review]
Love the automation! I only have one comment for discussion.

The difference between change, feature, and improvement isn't super clear without the explanations. Perhaps they could be labeled something like "Breaking Change", "Feature Enhancement", and "Optimization Improvement". I'm open to suggestions, especially if they're less verbose than my ideas.

File: changelog/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+   For example:
+
+       ```release-note:bug
+
+   Valid modes are:
+
+    - `bug` - Any sort of non-security defect fix.
+    - `security` - Any fixes for security issues in previous versions.
+    - `change` - A change in the product that may require action or
```
Comment (aliciaclark1066) [inline changelog/README.md:15]: Update wording to reflect "breaking change"

**Review by aliciaclark1066 — COMMENTED** [review]
Added final comment below on readme.

File: changelog/README.md (hunk 1 above)
Comment (nwchandler) [reply changelog/README.md]: Whoops! Updated.

**Review by aliciaclark1066 — APPROVED** [review]

**Pull request merged**